### PR TITLE
XW-3318 | update font size in local nav, screen breakpoint changes

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3318
@Wikia/x-wing

### CODE REVIEW

File: style-guide/styles/wds-components/_global-footer.scss [hunk 1]
```diff
@@ -334,7 +334,7 @@
 		padding: 0;
 	}
 
-	@media #{$wds-breakpoint-l-down} {
+	@media #{$wds-breakpoint-xl-down} {
```
Comment (rybmat) [inline style-guide/styles/wds-components/_global-footer.scss]: @latata why did you change global footer styles?

Comment (latata) [inline style-guide/styles/wds-components/_global-footer.scss]: because I changed the breakpoint variable name, as I mentioned in description.

Comment (rybmat) [inline style-guide/styles/wds-components/_global-footer.scss]: ok, I missed that

**Comment by rogatty** [comment]
Make sure that http://fandomdesignsystem.com/#/base-styles/breakpoints is up to date.
